### PR TITLE
Add first draft

### PR DESCRIPTION
**Pull request by Dav1s-Ops**

### CODE REVIEW

**Review by Gabe-Torres — COMMENTED** [review]
Solid changes! This is good.

**Review by kcockett — CHANGES_REQUESTED** [review]
Please use '-' for unordered lists.  Refer to this for using Markdown:  https://www.markdownguide.org/cheat-sheet/#overview

**Review by Gabe-Torres — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+  - Code design pending..
+
+# Contributors
+
+## Will Zale
+  - LinkedIn: https://www.linkedin.com/in/william-zale-764b0921/
+  - GitHub: https://github.com/WZale
+
+## Gabriel Torres
+  - LinkedIn: https://www.linkedin.com/in/gabe-torres-74a515269/
```
Comment (WZale) [inline README.md:32]: Nice! Glad you could get the formatting right.

**Review by WZale — APPROVED** [review]

**Comment by WZale** [comment]
Nice! Looks great, thanks for taking care of this.

**Review by kcockett — APPROVED** [review]
Ok, changes look great!

**Pull request merged**